app/controllers/idv/usps_controller.rb

```diff
@@ -58,15 +58,15 @@ def failure
       redirect_to idv_usps_url unless performed?
     end
 
-    def pii
+    def pii(pii_params)
       hash = {}
-      update_hash_with_address(hash)
+      update_hash_with_address(hash, pii_params)
       update_hash_with_non_address_pii(hash)
       hash
     end
```

Comment:
WDYT of taking this time while we're here to rewrite `update_hash_with_non_address_pii`?

We could just make it return a new hash rather than update an existing one, then be like


```
pii = non_address_pii.merge(pii_params)
```

but maybe rename `pii_params` to `address_pii` or something